phonebook.code-workspace

```diff
@@ -43,11 +43,6 @@
     "typescript.preferences.quoteStyle": "single",
     "javascript.referencesCodeLens.enabled": true,
     "typescript.referencesCodeLens.enabled": true,
-    "prettier.jsxSingleQuote": true,
-    "prettier.singleQuote": true,
-    "prettier.tslintIntegration": true,
```

Comment:
You are sure to remove this?

Reply:
_"This setting has been deprecated. See: https://github.com/prettier/prettier-vscode#vscode-eslint-and-tslint-integration"_

Reply:
Ok, 👍